rest/system-config-get.rst

```diff
@@ -3,7 +3,8 @@ GET /rest/system/config
 
 .. deprecated:: v1.12.0
    This endpoint still works as before but is deprecated. Use :ref:`rest-config`
-   instead.
+   instead.  Also note that the ``pendingFolders`` and ``pendingDevices``
+   fields have been removed as of v1.FIXME.
```

Comment:
I wouldn't start documenting individual changes to the config. We didn't do that until now and I don't see us (consequently) doing it in the future. Having just a few mentioned might confuse and/or create wrong assumptions that it is documented. And having them in the config is not an error.

I just wanted to say that there should however be a deprecation note in users/config.rst, however the pending elements were never documented there.

Reply:
So what do we do about the examples?  Just remove the corresponding lines?

Is there an easy way to browse older versions of the deployed documentation? Like we know it from readthedocs for example.

Reply:
I'd just remove them.

Good question. I remember discussing that, probably when talking about alternative ways to organize/edit documentation. I don't think there's any way to select past version currently.

Reply:
There are the previews for every checked commit in a PR, but they vanish after some time.  Maybe the same mechanism could be used to deploy to a permanent location for each tag?

Reply:
> I'd just remove them.

Done.